### DISCUSSION TITLE
Question about calendar colors

### DISCUSSION THREAD

**Discussion by egoitzau** [Z-Hub/Z-Push#50] [General]
Hi!

Have you been able to get different calendar colors in Active sync?. It seems (seems but not sure) Android does not allow to use more than one color and when you have more than one calendar... that could become difficult to see...

By the way, I have color-blind users and it would be nice for them to be able to change at least the calendar color. Any ideas mates?.

Cheers!!

**Comment by matidau** [comment]
Heya,

I think that this is all from the Android and/or app side of things.

With a quick search there is apparently calendar colour picker apps that let you set this, haven't tried these myself.

Cheers,
Mat
